workitem/typegroup/typegroup.go

```diff
@@ -54,3 +55,14 @@ var Requirements0 = WorkItemTypeGroup{
 		workitem.SystemBug,
 	},
 }
+
+// Execution0 defines first level of typegroup Execution
+// This group has less priority than Requirements
+var Execution0 = WorkItemTypeGroup{
+	Group: GroupExecution,
+	Level: []int{2, 0},
```

Comment:
question: why does `Level: []int{2,0} mean` ? It should be good to explain what this means with a comment especially for external devs.